src/Deprecated/Engine/Items/BuildItemGroup.cs

```diff
@@ -16,6 +16,11 @@
 namespace Microsoft.Build.BuildEngine
 {
     /// <summary>
+    ///   <format type="text/markdown"><![CDATA[
+    ///      > [!CAUTION]
+    ///      > This class has been deprecated. Please use Microsoft.Build.Evaluation.ProjectCollection and related types from the Microsoft.Build assembly instead.
+    ///      ]]>
+    ///   </format>
```

Comment:
How does this show up in IDEs?

Reply:
This is how it currently shows on VS 2022
![image](https://github.com/dotnet/msbuild/assets/10743736/21839cff-5116-4ba4-b3d7-c5302baedf84)

Reply:
That's what I was worried about :-/ That's quite cluttered in the VS view.